taskbrowser/TaskBrowser.cpp

```diff
@@ -1450,6 +1616,30 @@ namespace OCL
             return;
         }
 
+        // Minor hack : since GlobalsRepository is nor a peer or a service therefore
+        // seen_dotmember() wouldn't work.
+        if ( comm.find("GlobalsRepository.") == 0  ) {
+            std::string::size_type pos = comm.find("GlobalsRepository.")+strlen("GlobalsRepository.");
+            std::string global_var_name = std::string(comm, pos, comm.length());
+            GlobalsRepository::shared_ptr globals = GlobalsRepository::Instance();
+
+            if ( globals->hasAttribute( global_var_name ) ) {
+                cerr << "Found value..."<<nl;
+                this->printResult( globals->getValue( comm )->getDataSource().get(), true );
+                cout << sresult.str()<<nl;
+                sresult.str("");
+                return;
+            }
+
+            if ( globals->hasProperty( global_var_name ) ) {
+                cerr << "Found value..."<<nl;
+                this->printResult( globals->properties()->find(global_var_name)->getDataSource().get(), true );
+                cout << sresult.str()<<nl;
+                sresult.str("");
+                return;
+            }
+        }
+
```

Comment:
This "hack" allows to print values of attributes and properties in the `GlobalsRepository`, but not to modify them or use their values in expressions.

Reply:
Using `GlobalsRepository` in expressions requires a patch in RTT scripting, see https://github.com/orocos-toolchain/rtt/pull/218/files#diff-9dec447992a6887a7248d74034d058bcR174. But the syntax would be different. I would vote for removing this hack here in favor of `printGlobals()` and the update of the `RTT::scripting::ValueParser`. Both should enable users to list and modify global properties.